engine/execution/testutil/fixtures.go

```diff
@@ -192,8 +192,12 @@ func CreateAccountsWithSimpleAddresses(
 		var addr flow.Address
 
 		for _, event := range tx.Events {
-			if event.EventType.ID() == string(flow.EventAccountCreated) {
-				addr = event.Fields[0].ToGoValue().([8]byte)
+			if event.Type == flow.EventAccountCreated {
+				data, err := jsoncdc.Decode(event.Payload)
+				if err != nil {
+					return nil, errors.New("error decoding events")
+				}
+				addr = flow.Address(data.(cadence.Event).Fields[0].(cadence.Address))
```

Comment:
Use `flow.AccountCreatedEvent(event).Address()` instead

Reply:
`AccountCreatedEvent` doesn't exist in flow, I guess flowsdk has some functionality

Reply:
but I'll add it.

Reply:
hmm, I can't add it, it adds circular dependencies

Reply:
Ah, I thought this was declared in the models, nevermind then